### PR TITLE
Check for assets config before adding precompile. Fixes hotwired/stimulus-rails#48

### LINKED ISSUE #48: Fails when used in project with no sprockets
I have a new project configured with webpacker and no sprockets. Stimulus rails fails to load as it is looking for a configuration option that is not available.

### PR DESCRIPTION
**Pull request by coder2000**

### CODE REVIEW

**Comment by coder2000** [comment]
Tests completed normally when run locally.

File: lib/stimulus/engine.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ class Engine < ::Rails::Engine
     config.autoload_once_paths = %w( #{root}/app/helpers )
 
     initializer "stimulus.assets" do
-      Rails.application.config.assets.precompile += %w( importmap.json stimulus/manifest )
+      Rails.application.config.respond_to?(:assets) do
+        Rails.application.config.assets.precompile += %w( importmap.json stimulus/manifest )
+      end
```
Comment (sandrasi) [inline lib/stimulus/engine.rb:10]: This is the actual fix, right? Can only be this included in the PR and the rest reverted?

Comment (coder2000) [reply lib/stimulus/engine.rb:10]: Maybe, but as mime_magic was removed as a dependency in Rails an update to 6.1.3.1 is required and it shouldn't affect 2.7.2 but Ruby 3.0.0 requires the addition of rexml to the Gemfile.

**Comment by dhh** [comment]
Merged another fix. Thanks for working on this.

**Pull request closed without merging**